### DISCUSSION TITLE
请问各位大佬，我这个是啥原因啊，我已经明明已经登陆openai账户和chatGPT，在亚马逊aws弄EC2前半部分设置完检查日志，查chatGPT登陆是否正常一直提示我登陆失败。

### DISCUSSION THREAD

**Discussion by tobby518** [lss233/chatgpt-mirai-qq-bot#1262] [General]
![微信图片_20231127062712](https://github.com/lss233/chatgpt-mirai-qq-bot/assets/151758382/c7ad6015-9ea0-43da-ab67-659502d07bbe)
请问各位大佬，我这个是啥原因啊，我已经明明已经登陆openai账户和chatGPT，在亚马逊aws弄EC2前半部分设置完检查日志，查chatGPT登陆是否正常，输入docker-compose logs -f --tail=100 检查日志，一直提示我登陆失败。我这个要怎么解决，麻烦给我说说，感谢

**Comment by sanbei101** [comment]
The ladder is locked, so let's get another one

**Reply by tobby518** [reply]
梯子锁住了？我用的veee+换其他线，也不行啊，大佬
